src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DAC/PX1027/DacForbiddenFieldsTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Acuminator.Analyzers;
+using Acuminator.Analyzers.FixProviders;
+using Acuminator.Tests.Helpers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using TestHelper;
+using Xunit;
+
+namespace Acuminator.Tests
+{
+    public partial class DacForbiddenFieldsTests : CodeFixVerifier
+    {
+        [Theory]
+        [EmbeddedFileData(@"Dac\PX1027\Diagnostics\DacForbiddenFields.cs")]
+        public virtual void TestDacWithForbiddenFields(string source) =>
+            VerifyCSharpDiagnostic(source,
+                //DAC part 1
+                CreatePX1027ForbiddenDacFieldDiagnosticResult(line: 13, column: 31, fieldName: "companyId"),
+                CreatePX1027ForbiddenDacFieldDiagnosticResult(line: 17, column: 23, fieldName: "CompanyID"),
+                CreatePX1027ForbiddenDacFieldDiagnosticResult(line: 27, column: 31, fieldName: "deletedDatabaseRecord"),
+                CreatePX1027ForbiddenDacFieldDiagnosticResult(line: 30, column: 23, fieldName: "DeletedDatabaseRecord"),
+                CreatePX1027ForbiddenDacFieldDiagnosticResult(line: 39, column: 31, fieldName: "companyMask"),
+                CreatePX1027ForbiddenDacFieldDiagnosticResult(line: 42, column: 23, fieldName: "CompanyMask"));
+                //DAC part 2
+               /* CreatePX1027ForbiddenDacFieldDiagnosticResult(line: 49, column: 31, fieldName: "deletedDatabaseRecord"),
```

Comment:
@saloponov, please don't leave the commented code. If you don't need it then you should remove it